### ISSUE TITLE
[1.0.0] "TypeError: an integer is required" during _store_webroot_validation

### ISSUE THREAD

**Issue by hartwork** [costela/wile#32]
Please excuse my brevity. I ran by this traceback using wile 1.0.0 today:
```
INFO:wile.cert:storing validation of WebrootTuple(remote_user=None, remote_host=None, remote_port=None, path='/var/www/xxxxxxxxxxxx/htdocs')
Traceback (most recent call last):
  File "/usr/lib/python-exec/python2.7/wile", line 11, in <module>
    load_entry_point('wile==0.0.0', 'console_scripts', 'wile')()
  File "/usr/lib64/python2.7/site-packages/wile/__init__.py", line 112, in main
    return wile(obj=LazyClient())
  File "/usr/lib64/python2.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib64/python2.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/lib64/python2.7/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/lib64/python2.7/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/lib64/python2.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/lib64/python2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/usr/lib64/python2.7/site-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/usr/lib64/python2.7/site-packages/wile/cert.py", line 112, in request
    authzrs = _generate_validation_requests(domain_list, webroot_list, ctx, regr, ssh_private_key)
  File "/usr/lib64/python2.7/site-packages/wile/cert.py", line 181, in _generate_validation_requests
    chall_validation)
  File "/usr/lib64/python2.7/site-packages/wile/cert.py", line 273, in _store_webroot_validation
    with chall_mod.open(chall_path, 'wb') as outf:
TypeError: an integer is required
ERROR: cert sync failed
```

**Comment by hartwork** [comment]
Found this to be a regression from fd56c945ebf600af1f525bf859d5d4be996a21ae, so releases prior to 1.0.0 should not be affected.

**Comment by costela** [comment]
closed by #33
